test/conftest.py

```diff
@@ -37,24 +29,28 @@
 with open('test/config.yaml', 'r') as yaml_file:
     config = yaml.load(yaml_file)
 
-@pytest.fixture()
+@pytest.fixture(scope="class")
 def setup(request):
     return config
 
-def pytest_configure():   
+def pytest_configure():
     # Generate data
-    producer = KafkaProducer(bootstrap_servers=config["kafka_broker_url"], value_serializer=lambda v: json.dumps(v).encode('utf-8'))
-    msg = {'foo': 'bar'} 
+    now = datetime.now()
+    msg = {'timestamp': str(datetime.timestamp(now))}
+    producer = KafkaProducer(bootstrap_servers=config["kafka_broker_url"],
+                             value_serializer=lambda v: json.dumps(v).encode('utf-8'))
     producer.send(config["kafka_topic"], msg)
     producer.flush()
+    config['timestamp'] = msg["timestamp"]
 
     # Launch all connectors for tests
     for param in connect_params:
         create_kafka_connector(config, param)
     # wait for data to be ingested to Splunk
-    time.sleep(60)
+    time.sleep(100)
```

Comment:
Why are you increasing the sleep? Will this resolve flakiness of missing data?

Reply:
the tests were failing, i thought there wasn't enough time. but i revers it back to 60s now